src/openrct2-ui/windows/Ride.cpp

```diff
@@ -885,7 +885,7 @@ static void WindowRideDrawTabImage(rct_drawpixelinfo* dpi, rct_window* w, int32_
  */
 static void WindowRideDrawTabMain(rct_drawpixelinfo* dpi, rct_window* w)
 {
-    WidgetIndex widgetIndex = WIDX_TAB_1 + WINDOW_RIDE_PAGE_MAIN;
+    WidgetIndex widgetIndex = WIDX_TAB_1 + static_cast<int32_t>(WINDOW_RIDE_PAGE_MAIN);
```

Comment:
Does it complain about these as well? That’s a bit add, as these are not even bitwise operations.

Reply:
Yes, in this PR are only violations reported when switching to C++20. It's probably easy to test the sum on the godbolt link I put at the top, but it's impossible to use godbolt on my phone.

It might also be that my description is misleading and it's not just bitwise operation, but any arithmetic operation between two different enumerations.